app/com/linkedin/drelephant/ElephantRunner.java

```diff
@@ -179,7 +176,10 @@ public void run() {
         logger.info(String.format("Analyzing %s", analysisName));
         AppResult result = _analyticJob.getAnalysis();
         result.save();
-        logger.info(String.format("Analysis of %s took %sms", analysisName, System.currentTimeMillis() - analysisStartTimeMillis));
+        long processingTime = System.currentTimeMillis() - analysisStartTimeMillis;
+        logger.info(String.format("Analysis of %s took %sms", analysisName, processingTime));
+        MetricsController.setJobProcessingTime(processingTime);
```

Comment:
sure. I will take  a look later to see how i can add the normalized
metrics. Having both makes sense. So, go ahead.

On Tue, Aug 2, 2016 at 12:38 PM, Paul Bramsen <notifications@github.com>
wrote:

> In app/com/linkedin/drelephant/ElephantRunner.java
> <https://github.com/linkedin/dr-elephant/pull/126#discussion_r73103898>:
>
> > @@ -179,7 +176,10 @@ public void run() {
> >          logger.info(String.format("Analyzing %s", analysisName));
> >          AppResult result = _analyticJob.getAnalysis();
> >          result.save();
> > -        logger.info(String.format("Analysis of %s took %sms", analysisName, System.currentTimeMillis() - analysisStartTimeMillis));
> > +        long processingTime = System.currentTimeMillis() - analysisStartTimeMillis;
> > +        logger.info(String.format("Analysis of %s took %sms", analysisName, processingTime));
> > +        MetricsController.setJobProcessingTime(processingTime);
>
> @shankar37 <https://github.com/shankar37> I think leaving it at just
> processingTime makes the most sense here.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/linkedin/dr-elephant/pull/126/files/187bc5e87645c61370d936ceb4cfdf3f96ec9a3c#r73103898>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMlRIzWr3pMIGuP4d85bobH5yiWECxP4ks5qbu0DgaJpZM4JaScd>
> .
>